storage/registers.go

```diff
@@ -0,0 +1,35 @@
+package storage
+
+import (
+	"io"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// Todo: Update interface after https://github.com/onflow/flow-go/pull/4656/ gets merged
+
+// Registers defines methods for the register index.
+type Registers interface {
+	RegisterReader
+	RegisterWriter
+	io.Closer
```

Comment:
I don't think the Registers module need to handle the Close method. 

Registers might not be the only data stored in pebble, if let's say, chunk data pack is also stored in pebble, then ChunkDataPacks would have to provide the same Close method?

The Close method is a database level method, so we can let the database creator hold the pebble db instance and call it.

For instance, the FlowNodeBuilder is responsible for creating the badger DB instance, as well as closing it during shut down.
https://github.com/onflow/flow-go/blob/master/cmd/scaffold.go#L890

Reply:
I believe this is outdated, I already removed the close for these reasons, but @koko1123 probably needs to re-merge the branch.